docs/content/cache.md

```diff
@@ -0,0 +1,309 @@
+---
+title: "Cache"
+description: "Rclone docs for cache remote"
+date: "2017-09-03"
+---
+
+<i class="fa fa-archive"></i> Cache
+-----------------------------------------
+
+The `cache` remote wraps another existing remote and stores file structure
+and its data for long running tasks like `rclone mount`.
+
+To get started you just need to have an existing remote which can be configured
+with `cache`.
+
+Here is an example of how to make a remote called `test-cache`.  First run:
+
+     rclone config
+
+This will guide you through an interactive setup process:
+
+```
+No remotes found - make a new one
+n) New remote
+r) Rename remote
+c) Copy remote
+s) Set configuration password
+q) Quit config
+n/r/c/s/q> n
+name> remote
+Type of storage to configure.
+Choose a number from below, or type in your own value
+...
+ 5 / Cache a remote
+   \ "cache"
+...
+Storage> 5
+Remote to cache.
+Normally should contain a ':' and a path, eg "myremote:path/to/dir",
+"myremote:bucket" or maybe "myremote:" (not recommended).
+remote> local:/test
+The size of a chunk. Lower value good for slow connections but can affect seamless reading.
+Default: 5M
+Choose a number from below, or type in your own value
+ 1 / 1MB
+   \ "1m"
+ 2 / 5 MB
+   \ "5M"
+ 3 / 10 MB
+   \ "10M"
+chunk_size> 2
+How much time should object info (file size, file hashes etc) be stored in cache. Use a very high value if you don't plan on changing the source FS from outside the cache.
+Accepted units are: "s", "m", "h".
+Default: 5m
+Choose a number from below, or type in your own value
+ 1 / 1 hour
+   \ "1h"
+ 2 / 24 hours
+   \ "24h"
+ 3 / 24 hours
+   \ "48h"
+info_age> 2
+How much time should a chunk (file data) be stored in cache.
+Accepted units are: "s", "m", "h".
+Default: 3h
+Choose a number from below, or type in your own value
+ 1 / 30 seconds
+   \ "30s"
+ 2 / 1 minute
+   \ "1m"
+ 3 / 1 hour and 30 minutes
+   \ "1h30m"
+chunk_age> 3h
+How much time should data be cached during warm up.
+Accepted units are: "s", "m", "h".
+Default: 24h
+Choose a number from below, or type in your own value
+ 1 / 3 hours
+   \ "3h"
+ 2 / 6 hours
+   \ "6h"
+ 3 / 24 hours
+   \ "24h"
+warmup_age> 3
+Remote config
+--------------------
+[test-cache]
+remote = local:/test
+chunk_size = 5M
+info_age = 24h
+chunk_age = 3h
+warmup_age = 24h
+```
+
+You can then use it like this,
+
+List directories in top level of your drive
+
+    rclone lsd test-cache:
+
+List all the files in your drive
+
+    rclone ls test-cache:
+
+To start a cached mount
+
+    rclone mount --allow-other test-cache: /var/tmp/test-cache
+
+### Write Support ###
+
+Writes are supported through `cache`.
+One caveat is that a mounted cache remote does not add any retry or fallback
```

Comment:
I originally did this but removed it because it basically duplicated the LowLevelRetry feature. If there is an issue with the cloud provider it ended up being retried a lot of times